server/src/utils.rs

```diff
@@ -235,11 +236,65 @@ impl TimePeriod {
     }
 }
 
+pub fn parse_storage_size(size: Option<String>) -> Option<u64> {
```

Comment:
We'll use `human-size` crate introduced in https://github.com/parseablehq/parseable/pull/585.